docs/spec/lightclient/verification/Blockchain_A_1.tla

```diff
@@ -75,7 +75,7 @@ LBT == [header |-> BT, Commits |-> {NT}]
 
 (* the header is still within the trusting period *)
 InTrustingPeriod(header) ==
-    now <= header.time + TRUSTING_PERIOD
+    now < header.time + TRUSTING_PERIOD
```

Comment:
Does this change come from the implementation or the English spec? Are those consistent?

Reply:
this comes from the implementation, as otherwise I get a testing error: spec behavior is different from what we have in the implementation. And the English spec contains this `trusted.Header.Time > now - trustingPeriod`, so I guess this means that the English spec and the implementation agree, but the TLA+ spec is wrong in that respect; could you please verify that it's true?

Reply:
will do.

Reply:
So now we do model-based testing as conformance test between English spec, TLA+, and the implementation. Amazing 😉